src/projetos/SugestaoPage.tsx

```
@@ -150,6 +139,30 @@ export function SugestaoPage() {
+        try {
+            const resultado = await UsuarioService.buscarPorEmail(email)
+            if (resultado) {
+                setProfessores(resultado)
+                form.setValue('professores', resultado.nome) // Exibe nome ao invés do email
+                toast.success('Professor válido!')
+            } else {
+                setProfessores(undefined)
+                form.setValue('professores', '')
+                toast.error('Professor inválido.')
+            }
+        } catch (e) {
```

ESLint

[error] Delete `····`

(prettier/prettier)